lambdas/ec2/reaper.py

```diff
@@ -155,28 +185,32 @@ def validate_lifetime_value(lifetime_value):
 
     Return a match object if a match is found; otherwise, return the None from the search method.
     """
-    search_result = re.search(r'^([0-9]+)(w|d|h)$', lifetime_value)
+    search_result = re.search(r'^([0-9]+)(w|d|h|m)$', lifetime_value)
     if search_result is None:
         return None
     toople = search_result.groups()
     unit = toople[1]
     length = int(toople[0])
     return (length, unit)
 
-def calculate_lifetime_delta(lifetime_tuple):
+def calculate_lifetime_delta(lifetime):
```

Comment:
I think that this still needs to be `lifetime_tuple`; it is the `lifetime_match` description below that is incorrect and should be updated.

Reply:
Ah yeah, good catch. Updated!